### ISSUE TITLE
Display filename in tag editor

### ISSUE THREAD

**Issue by darealshinji** [nightingale-media-player/nightingale-hacking#337]
Is it possible to display the filename and full path of a song when I edit its id3tags/metadata?
That would be really helpful when adding missing tags based on file or directory name.

**Comment by freaktechnik** [comment]
The advanced tab had a field for it, maybe it's hidden now as it wasn't editable?

**Comment by luisgmarine** [comment]
Yes, when I was working on the advance tab the source field was something that we removed since it was ready only.  You can view the closed issue here, https://github.com/nightingale-media-player/nightingale-hacking/issues/225.

We could potentially add all the fields again even if they are read only, but I would bring that up with some of the other developers.

**Comment by freaktechnik** [comment]
I think it makes sense to display the full path somewhere, though not necessarily as disabled textfield but just as a label.

Sometimes the path contain valuable information about the file...

**Comment by rsjtdrjgfuzkfg** [comment]
I agree wiht @freaktechnik, as a label it would probably be helpful, as a edit box I'd consider it misleading ("why can't I relocate the file?" / "what do I have to do to enable this field?") as long as we do not add an option to actually change the location (which may be nice for some people, imho).

**Comment by PanderMusubi** [comment]
Please display the full path. If the path is too long, let it scroll horizontally. Make sure the user can select and copy the path to use in it in another application. Editing is indeed a tricky one, for that is the "Show File" menu item. If you let the file be opened in a file browser such as nautilus, one can easily edit the file name if needed.

**Comment by freaktechnik** [comment]
My proposed solution for this would look like this:
![tvrqsgi2zcpkt688lctcrxz0qptu30xvn4c2i1oow-0](https://cloud.githubusercontent.com/assets/640949/8891437/bdc87d40-3329-11e5-9ad6-ee102ff95b1a.png)

This is some recycled code that is already in place I updated and slightly improved. I want to add a filetype field and default value for properties that can be empty. Is there a facility for property default values? (For example 0 for the counts, or "Never" in english for the dates etc.). Oh, and the size field needs a unit.
